packages/gatsby/src/bootstrap/__tests__/remove-stale-jobs.ts

```diff
@@ -56,31 +53,45 @@ describe(`remove-stale-jobs`, () => {
     expect(removeStaleJobs(state)).toMatchSnapshot()
     expect(internalActions.removeStaleJob).toHaveBeenCalledTimes(1)
     expect(internalActions.removeStaleJob).toHaveBeenCalledWith(`1234`)
-    expect(publicActions.createJobV2).not.toHaveBeenCalled()
+    expect(internalActions.createJobV2FromInternalJob).not.toHaveBeenCalled()
   })
 
   it(`should enqueue pending jobs`, () => {
     const data = {
       job: {
         inputPaths: [`/src/myfile.js`],
         contentDigest: `1234`,
-      },
-      plugin: {
-        name: `test`,
-        version: `1.0.0`,
+        plugin: { name: `test` },
       },
     }
 
     state.jobsV2.incomplete.set(`1234`, data)
 
     isJobStale.mockReturnValue(false)
+    // `enqueueJob` will be called internally and while mocked it just return undefined
```

Comment:
It's not really returning undefined right?
```suggestion
    // `enqueueJob` will be called internally and while mocked it just return a simple job result
```

Reply:
I just meant that we needed to `mockReturnValue` with a promise on `enqueueJob`, otherwise action creator would crash (cannot read `.then` of undefined)